README.md

```diff
@@ -1,5 +1,18 @@
 # The OpenEuropa Whitelabel theme
 
+## Paragraphs
+
+The paragraphs below are not yet themed therefore not recommended for usage:
+
+- Contextual navigation
+- Document
+- Fact
```

Comment:
move "Fact" in list with  "Listing item" since its the same logic;
remove rich text